linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/router_clients.js

```diff
@@ -23,9 +25,18 @@ var RouterClient = (function() {
   function getLatencyData(client, latencyKeys) {
     var latencyData = _.pick(client.metrics, latencyKeys);
 
-    return _.mapKeys(latencyData, function(value, key) {
+    var tableData = _.mapKeys(latencyData, function(value, key) {
       return key.split(".")[1];
     });
+
+    var chartData = _.map(latencyData, function(latency, metricName) {
+      return {
+        name: metricName,
+        delta: latency
+      }
+    });
+
+    return { tableData: tableData, chartData: chartData };
```

Comment:
You're iterating over latencyData twice here.  It's not as elegant, but you could save yourself some time if you generate both tableData and chartData in the same pass.  Something like:

```
var tableData = {};
var chartData = [];

_.forEach(latencyData, function(latency, metricName) {
  tableData[split(".")[1]] = metricName;
  chartData.push({name: metricName, delta: latency})
}

return { tableData: tableData, chartData: chartData };
```

... but with proper javascript conventions :)